### PR TITLE
Make obsolete nodes invisible

### PR DESCRIPTION
**Pull request by ZiyunShang**
### Purpose

Due to Dynamo Issue https://github.com/DynamoDS/Dynamo/issues/9591 , some nodes in DynamoRevit should be obsolete and set invisible.

### Reviewers

@AndyDu1985 @mjkkirschner 

### FYIs

@QilongTang

### CODE REVIEW

**Comment by QilongTang** [comment]
LGTM

File: src/Libraries/RevitNodes/AnalysisDisplay/FaceAnalysisDisplay.cs [hunk 1]
```diff
@@ -180,6 +181,7 @@ private void
         /// <param name="description">An optional analysis results description to show on the results legend.</param>
         /// <param name="unitType">An optional Unit type to provide conversions in the analysis results.</param>
         /// <returns>A FaceAnalysisDisplay object.</returns>
+        [NodeObsolete("SurfaceDataObsolete", typeof(Properties.Resources))]
```
Comment (mjkkirschner) [inline src/Libraries/RevitNodes/AnalysisDisplay/FaceAnalysisDisplay.cs]: Hi @ZiyunShang does this attribute also hide the nodes?

**Comment by ZiyunShang** [comment]
Hi @mjkkirschner , I have a new commit to move message to resource file. Could you help review it, please?

**Comment by mjkkirschner** [comment]
LGTM - please verify if the nodeObsolete attribute hides the nodes as well. - if not you can use `IsVisibleInDynamoLibrary(false)`

File: src/Libraries/RevitNodes/AnalysisDisplay/FaceAnalysisDisplay.cs (hunk 1 above)
Comment (ZiyunShang) [reply src/Libraries/RevitNodes/AnalysisDisplay/FaceAnalysisDisplay.cs]: Yes, they have been hidden.

**Pull request merged**